src/app/components/header/header.component.ts

```diff
@@ -51,6 +52,8 @@ export class AppHeaderComponent implements OnInit {
   @Output()
   toggleClicked = new EventEmitter();
 
+  @Input() expandedSidenav: Boolean = true;
```

Comment:
Unlike in Java, the TypeScript/JavaScript requires `boolean` ;)

Reply:
Thanks Denys. VScode suggested to remove  the type declaration. I think that is fine and it can calculate the boolean type from the assignment "true"  :)